apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -341,6 +387,7 @@ export default NextAuth({
           name: token.name,
           username: token.username as string,
           role: token.role as UserPermissionRole,
+          sessionTimeout: session.user?.sessionTimeout as number,
```

Comment:
Making sure we have the `sessionTimeout` value to use it when encoding the session token